vspace/src/main/java/edu/asu/diging/vspace/core/factory/impl/ExhibitionFactory.java

```diff
@@ -0,0 +1,23 @@
+package edu.asu.diging.vspace.core.factory.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.diging.vspace.core.factory.IExhibitionFactory;
+import edu.asu.diging.vspace.core.model.IExhibition;
+import edu.asu.diging.vspace.core.model.impl.Exhibition;
+import edu.asu.diging.vspace.core.services.impl.ExhibitionManager;
+
+@Service
+public class ExhibitionFactory implements IExhibitionFactory {
+
+	@Autowired
+	private ExhibitionManager exhibitManager;
+	@Override
+	public IExhibition createSpace(String exhibitID) {
+		if(exhibitID.equals("New"))
```

Comment:
the hard-coded string already gives you a hint that this is not an ideal solution. Couple of problems:
- if you need to decide by a passed id if something should be create or not, use null not a "random" string
- this method is a factory method for creating spaces, so all it should do is creating spaces (no retrieval of spaces form the db)
- since your manager class will most likely use the factory at some point, you have a circular dependency here, which is a big no no

Reply:
For #1, I have used empty string, I could not handle null pointer exception while creating Exhibition in ExhibitionManager.

Reply:
what do you mean by that?

Reply:
For the New exhibition dropdown option, I cannot set the option value to null because it throws error even before I can handle it in my backend. So I have set the option value to empty string instead of a random string.

Reply:
but this is in the exhibition factory, you can call that method with null, can't you?